src/components/Header.js

```diff
@@ -34,122 +37,185 @@ function Header({ linkAccount, printClass }) {
-                                      </a>
-                                    </li>
-                                    <li className="aide-metier">
-                                      <a className="rf-nav__link lien-aide" href={aideMetier} target="blank" rel="noreferrer"
-                                        onClick={() => {
-                                          setMenuAideShow(false);
-                                        }}>
-                                        Aide m&eacute;tier<br/>
-                                        <span className="sous-titre-lien">Missions et cadre de travail.</span>
-                                      </a>
-                                    </li>
-                                  </>
-                                  }
-                                  {role === 'structure_coop' &&
-                                    <li className="aide-coop">
-                                      <a className="rf-nav__link lien-aide" href={aideStructure} target="blank" rel="noreferrer"
-                                        onClick={() => {
-                                          setMenuAideShow(false);
-                                        }}>
-                                        Aide espace Coop<br/>
-                                        <span className="sous-titre-lien">Consulter la Foire aux Questions.</span>
-                                      </a>
-                                    </li>
-                                  }
-                                </ul>
-                              </div>
+                              <Link className="rf-nav__btn rf-custom-link" to="/a-propos" title="&Agrave; propos de votre espace Coop" >
+                                <i className="ri-compasses-2-fill"></i>&Agrave; propos
+                              </Link>
                             </li>
                           </ul>
-                        </div>
-                      </li>
-
-                      { linkAccount === 'noConnected' &&
-                        <li className="rf-shortcuts__item">
-                          <a href="/login" className="rf-link" target="_self">J&rsquo;ai d&eacute;j&Agrave; un compte</a>
                         </li>
-                      }
+                        <li className="rf-shortcuts__item header-aide">
+                          <div className="" role="navigation" aria-label="aide">
+                            <ul className="rf-nav__list">
+                              <li className="rf-nav__item">
+                                <button className="rf-nav__btn rf-custom-link"
+                                  aria-expanded={menuAideShow} aria-controls="menu-liens-aide" aria-current="true"
+                                  onClick={() => {
+                                    setMenuAideShow(!menuAideShow);
+                                    setMenuUserShow(menuUserShow ? !menuUserShow : menuUserShow);
```

Comment:
```suggestion
                                    setMenuUserShow(!menuUserShow);
```
Devrait marcher non ?